guides/api.md

```diff
@@ -1,7 +1,7 @@
 # Hubs Server API
 Reticulum includes a [GraphQL](https://graphql.org/) API that grants programmatic access to server resources. 
 
-Note: This API is currently in alpha testing and is not yet available for use. (Users cannot generate API Access Tokens.)
+Note: This API is currently in alpha testing and will likely experience frequent updates.
```

Comment:
I might just say, "This API is not yet stable, and may undergo breaking changes in future updates."